src/app/Main/Body/QueryPage/Results/TableResults/QueryPageContent-TableResultsLayout.component.tsx

```diff
@@ -31,8 +31,10 @@ export interface TableResultsProps {
      * attributes of the columns
      */
     columnNameValues: string[];
-    tableData: TableInterface
+    isSumRowCol: boolean;

```

Comment:
See comment on reject: This is the component that handles the show sum state, so the state should be defined here. It is a feature of your TalbleResults components. It would be another case if this state comes from the api or is shared with other main feature components, then it should be putted to a container, but this is not the case here.

Reply:
Done. Commit "Some more changes "

Reply:
Done. Commit "Some more changes"